demo/src/test/java/com/mongodb/jdbc/utils/TestUtils.java

```diff
@@ -0,0 +1,439 @@
+        System.out.println("Driver Version: " + m_dbMeta.getDriverVersion());
+        System.out.println("Driver Major Version: "
+                + m_dbMeta.getDriverMajorVersion());
+        System.out.println("Driver Minor Version: "
+                + m_dbMeta.getDriverMinorVersion());
+        System.out.println("Server product Name: "
+                + m_dbMeta.getDatabaseProductName());
+        System.out.println("Server product version: "
+                + m_dbMeta.getDatabaseProductVersion());
+        System.out.println("Server product Major version: "
+                + m_dbMeta.getDatabaseMajorVersion());
+        System.out.println("Server product Minor version : "
+                + m_dbMeta.getDatabaseMinorVersion());
+        System.out.println("supports Mixed Case Quoted Identifiers: "
+                + m_dbMeta.supportsMixedCaseQuotedIdentifiers());
+        System.out.println("supports Mixed Case Identifiers: "
+                + m_dbMeta.supportsMixedCaseIdentifiers());
+        System.out.println("stores UpperCase Quoted Identifiers: "
+                + m_dbMeta.storesUpperCaseQuotedIdentifiers());
+        System.out.println("stores LowerCase Quoted Identifiers: "
+                + m_dbMeta.storesLowerCaseQuotedIdentifiers());
+        System.out.println("stores Mixed Case Quoted Identifiers: "
+                + m_dbMeta.storesMixedCaseQuotedIdentifiers());
+        System.out.println("stores Upper Case Identifiers: "
+                + m_dbMeta.storesUpperCaseIdentifiers());
+        System.out.println("stores Lower Case Identifiers: "
+                + m_dbMeta.storesLowerCaseIdentifiers());
+        System.out.println("stores Mixed Case Identifiers: "
+                + m_dbMeta.storesMixedCaseIdentifiers());
+        System.out.println("get Identifier Quote String: "
+                + m_dbMeta.getIdentifierQuoteString());
+        System.out.println("is ReadOnly: " + m_dbMeta.isReadOnly());
+        System.out.println("supports ResultSet Concurrency: "
+                + m_dbMeta.supportsResultSetConcurrency(ResultSet.TYPE_FORWARD_ONLY,
+                ResultSet.FETCH_UNKNOWN));
+
+        System.out.println("get Identifier Quote String: "
+                + m_dbMeta.getJDBCMajorVersion());
+        System.out.println("get Identifier Quote String: "
+                + m_dbMeta.getTimeDateFunctions());
+        System.out.println("get Identifier Quote String: "
+                + m_dbMeta.getStringFunctions());
+        System.out.println("get Identifier Quote String: "
+                + m_dbMeta.getNumericFunctions());
+    }
+
+    @Test
+    void testGetUrl() throws Exception
+    {
+        System.out.println(m_dbMeta.getURL());
+    }
+
+    @Test
+    void testSetCatalog() throws SQLException
+    {
+        m_conn.setCatalog("test_set_catalog");
+        System.out.println(m_conn.getCatalog());
+    }
+
+    @Test
+    // Note : Cancel is not supported yet
```

Comment:
Is there a JIRA ticket we can link here?

Reply:
I think it should be enough to just add a `TODO` here so we can easily search for cases that need more investigation